includes/helper-functions.php

```diff
@@ -990,3 +990,18 @@ function edac_generate_summary_stat( string $item_class, int $count, string $lab
             <div class="edac-panel-number-label">' . $label . '</div>
         </li>';
 }
+
+/**
+ * Check if an element has an extension that matches the provided list.
+ *
+ * @since 1.15.0
+ *
+ * @param string $item A file path or URL to check.
+ * @param array  $extensions An array of extensions to check for.
+ *
+ * @return bool
+ */
+function is_item_using_matching_extension( string $item, array $extensions ): bool {
```

Comment:
Should we added some input validation?

```php
if ( ! is_string( $item ) || ! is_array( $extensions ) ) {
   return false;
}
```

Reply:
The typed params mean that the values here can only ever be a string and an array, so it's not needed inside the function. This check could never fail and return early.